docs/snippets/update/notify_support.md

```diff
@@ -1,6 +1,6 @@
 ## Notify support
 
 Please tell support that you have updated your installation.
-They update your support portal to match the new version.
+They update your service portal to match the new version.
 This ensures that you receive notifications about new maintenance releases and security advisories for the correct version.
-You can contact support at support@ibexa.co or through your [Support portal](https://support.ibexa.co).
+You can contact support at support@ibexa.co or through your [Service portal](https://support.ibexa.co).
```

Comment:
Capitalization inconsistency: Shouldn't "Service portal" be capitalized everywhere?